### ISSUE TITLE
How can I save video animation in Lottie new version with scale?

### ISSUE THREAD

**Issue by htluandc2** [airbnb/lottie-android#2248]
Hi there,
I'm creating an function to save animation in Lottie based on this [code](https://github.com/rogererill/LottieRecorderTest). But it's only run on old version of Lottie (lower 4.0.0) because it has an parameter from old version: 
```
init {
    lottieDrawable.scale = VIDEO_WIDTH_PX / lottieDrawable.intrinsicWidth
  }
```
Scale parameter is removed. How can I save my video by intrinsic size. Thank you.

**Comment by gpeal** [comment]
Try `setBounds(…)`

**Comment by KnotDaone** [comment]
Airbnb why y'all missing with me bro? You know what I'm talking about On Feb 20, 2023 12:35 PM, Gabriel Peal ***@***.***> wrote:
Closed #2248 as completed.
